app/src/main/java/com/hyphenate/chatuidemo/DemoHelper.java

```diff
@@ -563,8 +563,9 @@ private void registerMessageListener() {
                     if (!hasForegroundActivities()) {
                         if (message.getBooleanAttribute(Constant.MESSAGE_ATTR_IS_CALL_PUSH, false)) {
                             EMClient.getInstance().chatManager().getConversation(message.getFrom()).removeMessage(message.getMsgId());
-                        }else{
-                            getNotifier().onNewMsg(message);
+                        } else {
+                            // FIXME: conflict with group push notification if it's turned off, push still come in when phone is unlocked as the app is woken up by the phone to run in the background
+//                            getNotifier().onNewMsg(message);
```

Comment:
this will disable all new message notification  @jerrywu12 , maybe you can check message type here to just ignore group message.

Reply:
@githubzhaoliang so sounds like this is a bug that we need to solve.